.pre-commit-config.yaml

```diff
@@ -17,7 +17,11 @@
   sha: v2.1.1
   hooks:
   - id: pydocstyle
-    exclude: blog/migrations
+    exclude: >
+     (?x)^(
+         blog/migrations/.*|
+         blog/tests/__init__.py
```

Comment:
Don't do this. Add a docstring for tests package there.

Reply:
Fixed.